### PR TITLE
zurzeit-proposal-submission2

### PR DESCRIPTION
**Pull request by zurzeit**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: project/zurzeit/README.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Basic Information
+=================
+
+My repository: <https://github.com/zurzeit/SpinBall.git>
+
+Problem to Solve
+================
+
+As we watch the baseball game or table tennis game, we can see the spinning ball moves with a tricky trajectory. That is  because the Magnus effect gives an additional force to the ball. As the result, the ball drops rapidly.
```
Comment (yungyuc) [inline project/zurzeit/README.rst:15]: This looks interesting.

**Review by yungyuc — CHANGES_REQUESTED** [review]
Please add your project to the list in the readme file in the `project` root directory.

**Comment by yungyuc** [comment]
Leave a message when you have the change done.  BTW, a PR should not use a branch that has the identical name to the target branch.  It's ok this time, but try not to do it in the future.

**Comment by zurzeit** [comment]
I upload my proposal folder and add my proposal to the proposal list in
README.rst. However, github says that the branch can't merge automatically
when I launch a PR. I am wondering what is the conflict that I should fix.

   - my new PR: https://github.com/yungyuc/nsdhw_22au/pull/242

Thank you in advance!

Yung-Yu Chen ***@***.***> 於 2022年11月3日 週四 上午9:00寫道：

**Pull request closed without merging**